deploy/helm/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
This changes how dependencies are stored on disk. And we depend on Helm 3.x anyway.